packages/react-instantsearch-dom/src/widgets/__tests__/DynamicWidgets.test.tsx

```diff
@@ -0,0 +1,357 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { ErrorBoundary } from 'react-error-boundary';
+import {
+  HierarchicalMenu,
+  Pagination,
+  connectRefinementList,
+  Panel,
+  InstantSearch,
+} from '../..';
+import DynamicWidgets from '../DynamicWidgets';
+
+const EMPTY_RESPONSE = {
+  results: [
+    {
+      hits: [],
+      nbHits: 0,
+      page: 0,
+      nbPages: 0,
+      hitsPerPage: 20,
+      exhaustiveNbHits: true,
+      query: '',
+      queryAfterRemoval: '',
+      params:
+        'highlightPreTag=%3Cais-highlight-0000000000%3E&highlightPostTag=%3C%2Fais-highlight-0000000000%3E&query=&facets=%5B%5D&tagFilters=',
+      index: 'instant_search',
+      processingTimeMS: 2,
+    },
+  ],
+};
+
+const createSearchClient = () => ({
+  search: jest.fn(() => Promise.resolve(EMPTY_RESPONSE)),
+  searchForFacetValues: jest.fn(() => Promise.resolve({})),
+});
+
+const RefinementList = connectRefinementList(
+  ({ attribute }) => `RefinementList(${attribute})`
+);
+
```

Comment:
Wondering if the whole test suite makes sense here since we rely on the core `DynamicWidgets` which is also part of this repo.

Reply:
So I understand it right, you suggest to only test the class part in the dom tests?

Reply:
`className`, and that it forwards the props and the children to the core component maybe?